cpp/modmesh/pilot/R3DWidget.hpp

```diff
@@ -69,9 +73,26 @@ class RScene
 
     void setFirstPersonCameraController() { setCameraController(new RFirstPersonCameraController(this)); }
 
+    void setBoundingBox(QVector3D buttom_left, QVector3D top_right)
```

Comment:
Move setter and getter function of bounding box from `RVertices` and `RLines` to `RScene`. 

Sine `RVertices` and `RLines` would set `RScene` as their parent, I decide to calculate bounding box in `RVertices` and `RLines` and then pass the value to their parent.

Reply:
This is a good idea to add `setBoundingBox()`.  Please make sure there is not a function named like this in the Qt parent class, and make the argument to be `QVector3D const &`, instead of pass by value (`QVector3D`).